src/core/model_config.proto

```diff
@@ -442,6 +442,27 @@ message ModelOptimizationPolicy
     bool graphs = 1;
   }
 
+  //@@
+  //@@  .. cpp:var:: message Tensorflow
+  //@@
+  //@@     TensorFlow-specific optimization settings.
+  //@@
+  message Tensorflow
```

Comment:
Can we just use a new accelerator instead of making a TF specific message?

Reply:
This can be moved to accelerator section, perhaps in `gpu_execution_accelerator` with the name `gpu_io`?

Reply:
Sounds good